UI/SavedataScreen.cpp

```diff
@@ -0,0 +1,364 @@
+// Copyright (c) 2013- PPSSPP Project.
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 2.0 or later versions.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License 2.0 for more details.
+
+// A copy of the GPL 2.0 should have been included with the program.
+// If not, see http://www.gnu.org/licenses/
+
+// Official git repository and contact information can be found at
+// https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
+
+#include <algorithm>
+
+#include "base/colorutil.h"
+#include "base/timeutil.h"
+#include "gfx_es2/draw_buffer.h"
+#include "i18n/i18n.h"
+#include "math/curves.h"
+#include "util/text/utf8.h"
+#include "ui/ui_context.h"
+#include "ui/view.h"
+#include "ui/viewgroup.h"
+#include "UI/SavedataScreen.h"
+#include "UI/MainScreen.h"
+#include "UI/GameInfoCache.h"
+#include "UI/ui_atlas.h"
+#include "UI/PauseScreen.h"
+
+#include "Common/FileUtil.h"
+#include "Core/Host.h"
+#include "Core/Config.h"
+#include "Core/SaveState.h"
+#include "Core/System.h"
+
+class SavedataButton;
+
+std::string GetFileDateAsString(std::string filename) {
+	tm time;
+	if (File::GetModifTime(filename, time)) {
+		char buf[256];
+		// TODO: Use local time format? Americans and some others might not like ISO standard :)
```

Comment:
We actually have a setting for this I think, although it could impact games so I dunno if it's the right thing to use.

-[Unknown]